src/main/java/util/Storage.java

```diff
@@ -0,0 +1,155 @@
+package util;
+
+import util.task.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file
+ * */
+public class Storage {
+    private String filePath;
+
+    /**
+     * Constructor for Storage class.
+     *
+     * @param filePath File path string for Duke's data.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads save file contents (if any).
+     * https://nus-cs2103-ay2021s1.github.io/website/book/cppToJava/misc/fileAccess/
+     *
+     * @return Array list of Tasks deciphered from save file.
+     * @throws FileNotFoundException  If no save file detected.
+     */
+    public ArrayList<Task> loadFileContents() throws FileNotFoundException {
+        // Initialize list to be returned
+        ArrayList<Task> list = new ArrayList<>(100);
```

Comment:
naming of list could be more specific?

Reply:
Agree with this. Perhaps consider renaming it to something more specific?